playbooks/group_vars/prometheus/public.yml

```diff
@@ -88,6 +88,46 @@ prometheus_rules:
       annotations:
         summary: "Instance [[ $labels.node ]] ([[ $labels.instance ]]) disk is filling up."
         description: "Instance disk may fill up {{ PROMETHEUS_ALERT_DISK_SPACE_THRESHOLD }}% in 4 hours."
+  - name: NodeRules_v2
+    # We created this to support alerts for nodes that has node exporter version > 0.15 
+    rules:
+    - alert: "cpu_usage_high"
```

Comment:
@karimelghazouly Alerts that you manually defined on the prometheus server have a `_v2` suffix (`cpu_usage_high_v2`, `low_memory_v2`, ...). I guess we have to add the suffix here as well, so that the alerts don't clash with the old rules?